docs/interoperability.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Using Enact with third-party libaries
+---
+
+## Overview
+
+`Enact` is a fantastic way to build apps. By default we give you a template using our `Moonstone`
+library. `Moonstone` is a library designed for TVs. However, if you're not looking to create TV
+apps, you can easily use `Enact` to build applications for desktop and mobile. You can mix and match
+third-party libraries to use with `Enact` easily. Alternatively, it is also straightforward if you
+just want to use `Enact` components in your own `React` app.
+
+## Using Third-Party Components inside of Enact
+When creating an app using `enact create`, it creates an `Enact` boilerplate that setups for a
+`Moonstone` styled app. However if you want to use something outside of `Moonstone` like
+`material-ui`, `reactstrap`, or `react-router` just use `npm install`.
+
+You can include components just like you normally would by using import.
+
+```JavaScript
+import { Button as BSButton} from 'reactstrap';
```

Comment:
Is `BSButton` the recommended way for `reactstrap`?  It seems odd to me, but I don't know `reactstrap`.

Reply:
Yeah I'll switch it back to button